README.md

```diff
@@ -79,6 +79,13 @@ in the [`span-stacktrace`](https://github.com/open-telemetry/opentelemetry-java-
 Experimental runtime metrics are enabled by default.
 Set `otel.instrumentation.runtime-telemetry.emit-experimental-telemetry` to `false` to disable them.
 
+
+### Metric Temporality
+
+Elasticsearch and Kibana work best with metrics provided in delta-temporality.
+Therefore, the EDOT Java changes sets the default value of `otel.exporter.otlp.metrics.temporality.preference` to `DELTA`.
```

Comment:
```suggestion
Therefore, the EDOT Java changes the default value of `otel.exporter.otlp.metrics.temporality.preference` to `DELTA`.
```

We also need to document this in https://github.com/elastic/opentelemetry/blob/main/docs/_edot-sdks/java/features.md, while not being a "feature" by itself we need to document that we have a different default here.